websocket-server/tsconfig.json

```diff
@@ -11,6 +11,7 @@
     "forceConsistentCasingInFileNames": true,
     "resolveJsonModule": true,
     "sourceMap": true,
+    "typeRoots": [ "./node_modules/@types" ],
```

Comment:
This is more restrictive than the default, which already includes this folder:

https://www.typescriptlang.org/tsconfig/#typeRoots


```suggestion
```